remote/src/test/scala/org/apache/pekko/remote/artery/tcp/SecureRandomFactorySpec.scala

```diff
@@ -75,7 +75,7 @@ abstract class SecureRandomFactorySpec(alg: String) extends PekkoSpec {
 
         val compressed = baos.toByteArray
         // random data should not be compressible
-        // Another reproducer of https://doc.akka.io/docs/akka/current/security/2018-08-29-aes-rng.html
+        // Another reproducer of https://pekko.apache.org/docs/pekko/current/security/2018-08-29-aes-rng.html
```

Comment:
revert this - we are not keeping the akka security docs in the pekko doc clone - so best to link this to the akka docs